designs/developer/operator-pipeline-46/Failure Details/index.md

```diff
@@ -0,0 +1,29 @@
+---
+parent: Developer
+version: 4.6
+---
+
+# Display PipelineRun failure details
+
+## Description 
+
+Displaying PipelineRun failure details will help users to easily understand why a PipelineRun has failed so that they can investigate the issues faster.
+
+## Failure on task
+
+The message above the log box states Failure on task "x" - check logs for details. The log box would show the last few lines of the failed TaskRun pod. The log box would be 5 lines with a horizontal scrollbar.
+
+![failureontask](img/image3.png)
```

Comment:
I think maybe the images are located in a folder called `images` but this link is expecting them to be in `img`, so they appear broken to me.

Reply:
Thank you @itsptk fixed it now!